titan-web-client/src/lib/styles/typography.js

```diff
@@ -30,15 +30,15 @@ export function createTypography (typography = {}) {
       fontFamily: DEFAULT_FONT_FAMILY,
       fontSize: DEFAULT_FONT_SIZE,
       lineHeight: 1,
-      body: createElementStyles(FONT_WEIGHT_REGULAR, 1, 1, TEXT_CASE_REGULAR),
-      h1: createElementStyles(FONT_WEIGHT_REGULAR, 1.6, 1.8, TEXT_CASE_CAPITALIZE),
-      h2: createElementStyles(FONT_WEIGHT_REGULAR, 1.5, 1.6, TEXT_CASE_CAPITALIZE),
-      h3: createElementStyles(FONT_WEIGHT_REGULAR, 1.4, 1.4, TEXT_CASE_CAPITALIZE),
-      h4: createElementStyles(FONT_WEIGHT_LIGHT, 1.3, 1.3, TEXT_CASE_REGULAR),
-      h5: createElementStyles(FONT_WEIGHT_LIGHT, 1.25, 1.25, TEXT_CASE_REGULAR),
-      h6: createElementStyles(FONT_WEIGHT_LIGHT, 1.25, 1.25, TEXT_CASE_REGULAR),
+      h1: createElementStyles(FONT_WEIGHT_REGULAR, 1.8, 2, TEXT_CASE_CAPITALIZE),
```

Comment:
I noticed the fonts were working by accident, as we had two theme providers fighting over each other. After removing the styled components theme provider, I had to adjust the fonts.